pkg/ccl/importccl/exportcsv_test.go

```diff
@@ -304,6 +304,13 @@ func validateParquetFile(t *testing.T, file string, truthRows [][]interface{}) e
 
 		t.Logf("\n Record %v:", count)
 		for i := 0; i < len(cols); i++ {
+			if truthRows[count][i] == nil {
+				// if we expect a null value, the row created by the parquet reader will not have the
+				// associated column
```

Comment:
[nit] I think most parts of the code base try to keep comments punctuated where possible:
```suggestion
				// If we expect a null value, the row created by the parquet reader will not have the
				// associated column.
```

Reply:
Done.